docs/topics/Gateway.md

```diff
@@ -691,6 +700,29 @@ A user's presence is their current state on a guild. This event is sent when a u
 >info
 >The streaming type currently only supports Twitch. Only `https://twitch.tv/` urls will work.
 
+###### Game Timestamps
+
+| Field | Type | Description |
+|-------|------|-------------|
+| start? | float/int | Epoch seconds of start |
+| end? | float/int | Epoch seconds of end |
+
+###### Game Party
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id? | string | The ID, relevant for joining the party |
+| size | Array containing the current and maximum size | First element is the current size, second the max `[2, 5]` |
```

Comment:
Let's be consistent with the type column and call it `array of two integers` and let description do the clarifying